misc/test-client-rust/src/main.rs

```diff
@@ -107,6 +107,26 @@ struct CliArgs {
     /// Two-factor authentication recovery key
     #[arg(long)]
     twofa_recovery_key: Option<String>,
+
+    /// List all user keys
+    #[arg(long)]
+    list_keys: bool,
+
+    /// Store or update a user key
+    #[arg(long)]
+    store_key: bool,
```

Comment:
Maybe we should replace this with `store_key: Option<String>` where the value is the key material. This would allow us to remove the `key_material` switch. Thoughts?